src/pages/RepoPage/CoverageOnboarding/ActivationBanner/ActivationBanner.spec.tsx

```diff
@@ -0,0 +1,97 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import { render, screen } from '@testing-library/react'
+import { graphql } from 'msw'
+import { setupServer } from 'msw/node'
+import { MemoryRouter, Route } from 'react-router-dom'
+
+import ActivationBanner from './ActivationBanner'
+
+jest.mock('./TrialEligibleBanner', () => () => 'TrialEligibleBanner')
+
+const queryClient = new QueryClient()
+
+const wrapper = ({ children }: { children: React.ReactNode }) => (
```

Comment:
Can you swap this to a `React.FC`, bit more consistent with the codebase.

```suggestion
const wrapper: React.FC<React.PropsWithChildren> = ({ children }) => (
```